godot-google-play-billing/build.gradle

```diff
@@ -0,0 +1,26 @@
+plugins {
+    id 'com.android.library'
+}
+
+android {
+    compileSdkVersion 29
+    buildToolsVersion "29.0.3"
+
+    defaultConfig {
+        minSdkVersion 18
+        targetSdkVersion 29
+    }
+
+    libraryVariants.all { variant ->
+        variant.outputs.all { output ->
+            output.outputFileName = "GodotGooglePlayBilling.${variant.name}.aar"
+        }
+    }
+
+}
+
+dependencies {
+    implementation "androidx.core:core:1.3.0"
```

Comment:
This dependency is different from the originals. If we can keep using the originals, let's try them first, otherwise we'll also need to add this as a `remote` dependency in the `.gdap` config file since the build template does not yet have this dependency.

Reply:
Sure :)